src/tree/index.tsx

```diff
@@ -0,0 +1,209 @@
+import { create, tsx, diffProperty } from '@dojo/framework/core/vdom';
+import { createICacheMiddleware } from '@dojo/framework/core/middleware/icache';
+import theme from '@dojo/framework/core/middleware/theme';
+import { RenderResult } from '@dojo/framework/core/interfaces';
+import Icon, { IconType } from '../icon';
+
+import * as css from '../theme/default/tree.m.css';
+import Checkbox from '../checkbox';
+
+export interface TreeNode {
+	content: string | RenderResult;
```

Comment:
We shouldn't be passing `RenderResult` as widget properties, they should only be children

Reply:
Replaced with `value: string` and a custom renderer via child on `Tree`.